tasks/reload-module.yml

```diff
@@ -8,3 +8,4 @@
   modprobe:
     name: "{{ nested_virtualization_module_name }}"
     state: present
+    params: "nested=1"
```

Comment:
This role works in two modes, where you can enable or disable nesting.
I believe that adding this option discontinue such ability.

https://github.com/lukas-bednar/ansible-role-nested-virtualization/blob/master/README.md#example-playbook

Reply:
Should do the trick now.